GameData/RP-1/Contracts/Commercial Applications 1/FirstNavSat-CA.cfg

```diff
@@ -0,0 +1,125 @@
+CONTRACT_TYPE
+{
+	name = FirstNavSat-CA
+	title = First Navigational Satellite  // Based on Transit 1 through 4
+	group = CommApp
+	agent = Federation Aeronautique Internationale
+
+	description = <b>Program: Early Commercial Applications<br>Type: <color=green>Required</color></b><br><br>Through measuring the Doppler shift of radio signals transmitted from a satellite in a known orbit, it is possible for a receiver on the ground to establish their location, which would have many applications both civilian and military. To test the principle and develop receiver equipment, it is necessary to have a test navigational satellite in orbit. Be sure to include solar panels to generate power.<br>Historically, the first experiments in satellite navigation were made with Transit 1B (119 kg, Thor-Ablestar).
+
+	synopsis = Launch the first navigation satellite
+
+	completedMessage = Congratulations! The satellite is in orbit and we have already begun to receive Doppler curves.
+
+	sortKey = 410
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 40
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyCommApp
+	}
+
+	PARAMETER
+	{
+		name = FirstNavSat
+		type = VesselParameterGroup
+		title = First Navigational Satellite
+		define = FirstNavSat
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = HasNavSatPayload
+			type = HasResource
+			resource = NavSatPayload
+			minQuantity = 49.9
+			title = Have a NavSatPayload of at least 50 units on the craft
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = Orbit
+			type = Orbit
+			minInclination = 50
+			minPeA = 350000
+			maxEccentricity = 0.26
+			disableOnStateChange = true
+			title = Achieve orbit with a minimum inclination of 50 degrees and a minimum perigee of 350 km
```

Comment:
I think this doesn't need a title anymore? Because it's got a bunch of parameters.